src/js/tabs/pid_tuning.js

```diff
@@ -1758,6 +1780,27 @@ pid_tuning.initialize = function (callback) {
             context.clearRect(0, 0, canvasWidth, canvasHeight);
             context.beginPath();
             context.moveTo(0, canvasHeight);
+            if (throttleLimitType === THROTTLE_LIMIT_TYPES.CLIP){
+                const clippos = throttleLimitPercent <= midy
+                    ? getPosfromYBezier(throttleLimitPercent,canvasHeight,midyl, midy)
+                    : getPosfromYBezier(throttleLimitPercent,midy, midyl, 0);
+                if (throttleLimitPercent <= midyl){
+                    //first half
+                    lowerctrl = getQuadraticCurvePoint(0, canvasHeight, midxl, midyl, midx, midy, clippos/2);
+                    curveclip = getQuadraticCurvePoint(0, canvasHeight, midxl, midyl, midx, midy, clippos);
+                    context.quadraticCurveTo(lowerctrl.x, lowerctrl.y, curveclip.x, curveclip.y);
+                    context.moveTo(curveclip.x, curveclip.y);
+                    context.lineTo(canvasWidth, curveclip.y);
```

Comment:
You can put these lines after the if else block - as they are the same in both conditions

Reply:
Same for lines 1799 - 1801.

Reply:
Lines 1799 - 1801 won't be needed.